kvmagent/kvmagent/plugins/zbs_storage_plugin.py

```
@@ -39,6 +42,33 @@ def check_host_storage_connection(self, req):
 
         return jsonobject.dumps(rsp)
 
+    @kvmagent.replyerror
+    @bash.in_bash
+    def update_host_dependency(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = CheckHostStorageConnectionRsp()
+
+        yum_cmd = "export YUM0={};yum --enablerepo=* clean all && yum --disablerepo=* --enablerepo={},zstack-experimental-mn {} update {} -y"
+        yum_cmd = yum_cmd.format(releasever, cmd.zstackRepo, exclude, updates)
```

Ruff (0.14.5)

Undefined name `releasever`

(F821)